src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -220,65 +256,91 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
         cbsd_dump_data = []
         ppa_dump_data = []
         esc_sensor_dump_data = []
-        # STEP 5
-        self.assertContainsRequiredFields("FullActivityDump.schema.json", response)
         
         # step 6 and check   
         # download dump files and fill corresponding arrays
         for dump_file in response['files']:
             self.assertContainsRequiredFields("ActivityDumpFile.schema.json",
                                                dump_file[0])
+            downloaded_file = None
+            if dump_file['recordType'] != 'CoordinationEvent':                
+                downloaded_file = self._sas.DownloadFile(dump_file['url'])
+                datetime.strptime(downloaded_file['startTime'],\
+                                                     '%Y-%m-%dT%H:%M:%SZ')
+                datetime.strptime(downloaded_file['endTime'],\
+                                                     '%Y-%m-%dT%H:%M:%SZ')
             if dump_file['recordType'] ==  'cbsd':
-                cbsd_dump_data.append(self._sas.DownloadFile(dump_file['url'])['recordData'])   
+                cbsd_dump_data.append(downloaded_file['recordData'])   
             elif dump_file['recordType'] ==  'esc_sensor':
-                esc_sensor_dump_data.append(self._sas.DownloadFile(dump_file['url'])['recordData'])
+                esc_sensor_dump_data.append(downloaded_file['recordData'])
             elif dump_file['recordType'] ==  'zone':
-                ppa_dump_data.append(self._sas.DownloadFile(dump_file['url'])['recordData'])
+                ppa_dump_data.append(downloaded_file['recordData'])
+            else:
+                self.assertEqual('CoordinationEvent', dump_file['recordType'])
         
         # verify the length of records equal to the inserted ones
         self.assertEqual(len(config['registrationRequests']), len(cbsd_dump_data))
         self.assertEqual(len(config['ppas']), len(ppa_dump_data))
-        self.assertEqual(len(config['esc_sensors']), len(esc_sensor_dump_data))
-        
-        # verify the schema of record and that the injected ppas exist in the dump files
-        for index, ppa in enumerate(config['ppas']):
-            ppa_record = [record for record in ppa_dump_data if record['id'].split("/")[-1] == ppa_ids[index]][0]
+        self.assertEqual(len(config['escSensors']), len(esc_sensor_dump_data))
+        # verify the schema of record and first two parts of PPA record Id  
+        for ppa_record in ppa_dump_data:
+            self.assertContainsRequiredFields("zoneData.schema.json", ppa_record)              
+            self.assertEqual(ppa_record['id'].split("/")[0], 'zone')
+            self.assertEqual(ppa_record['id'].split("/")[1], self._sas._sas_admin)
             del ppa_record['id']
-            self.assertContainsRequiredFields("zoneData.schema.json", ppa_record)
-            self.assertDictEqual(ppa, ppa_record)
+        # verify that the injected ppas exist in the dump files
```

Comment:
The PPA check below does not appear to implement the "polygon equivalence" we discussed earlier. If you plan to do this in a followup PR, please add a comment like

`
TODO: check that the PPAs overlap nearly entirely, rather than requiring exactly the same vertices.
`

Reply:
I understood that for the moment we will test the SAS to have the exact values and later we can update it to have the margin, which we need to specify then.
I updated added the comment, thank you.
If you think it is better to have this margin now, I can update the code :)

Reply:
Oops, sorry. I'm not always good at remembering conversation details so thanks for the reminder. The TODO is good for now -- I think it should be a relatively quick fix in a followup PR so I'm not too concerned.